Controller/NotifyHandler.php

```diff
@@ -197,6 +197,7 @@ private function _onNotificationSuccess($oResponse, $oBackendService, $sPaymentI
         $oOrder = oxNew(Order::class);
         if ($this->_loadOrder($oOrder, $sTransactionId) >= self::MAX_TIMEOUT_SECONDS) {
             $this->_oLogger->error('No order found for transactionId: ' . $sTransactionId);
+            ResponseHandler::saveTransaction($oResponse, $oOrder, $oBackendService);
```

Comment:
Please, make sure to only save transactions that cannot be related to orders **for POI/PIA** avoiding to affect any other payment method flows.

Reply:
👍